paper-input-container.html

```diff
@@ -417,6 +417,10 @@
       return Polymer.dom(this).querySelector(this._inputSelector);
     },
 
+    get __inputElementValue() {
```

Comment:
Private: a discerning choice. But! If you can't inherit from this thing (it is an element) isn't every protected method also a private method?